lib/core/output_sink/base_sink.rb

```diff
@@ -18,7 +18,7 @@ def ingest_multiple(_documents)
       end
 
       def delete(_id)
-        raise 'not_implemented'
+        raise 'not implemented'
```

Comment:
why not using `NotImplementedError` ?

Reply:
We had different ways to raise "not implemented" in the project, in this PR I just change it to be consistent within a file. I'll raise a different PR to change this throughout the project :)